<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GazeVisualizationToolbox</class>
 <widget class="QWidget" name="GazeVisualizationToolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>134</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>139</width>
    <height>93</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Toolbox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="configurationButton">
     <property name="text">
      <string>Repeat Configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showGazePosCheckBox">
     <property name="text">
      <string>Show Gaze Positions</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>Quit Application</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>GazeVisualizationToolbox</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
